game/effect.go

```diff
@@ -1,8 +1,49 @@
+/*
+	An Effect can be created by a spell, a triggered ability, or an
+	activated ability.
+
+	An Effect can be the property of a Card. For example, when the
+	Card is a Giant Growth, its effect property would be
+	&Effect{power:3, toughness:3}.
+
+	An Effect can also be the Morbid or Kicker property of a Card, to designate a
+	Effect that only happens under special circumstances.
+
+*/
+
 package game
 
 import ()
 
 type Effect struct {
-	Action *Action
-	Card   *Card
+
+	// from Card
+	CastingCost        *CastingCost // when an Effect is a kicker, it has a Cost
+	Hexproof           bool
+	Kicker             *Effect
+	Plus1Plus1Counters int
+	Power              int
+	Summon             *Card
```

Comment:
this sometimes points to a spawn token so that is why you want normal functions of cards like `NewCard` to work for spawn tokens